tests/foreman/ui/test_http_proxy.py

```diff
@@ -521,3 +521,38 @@ def test_positive_repo_discovery(setup_http_proxy, module_target_sat, module_org
                 product_name, settings.container.docker.repo_upstream_name
             )[0]['Name']
         )
+
+
+def test_authenticated_test_connection(target_sat, module_sca_manifest_org, default_location):
```

Comment:
```suggestion
def test_authenticated_test_connection(target_sat, module_org, default_location):
```
Let's avoid using the manifest workflow (create allocation, add subs, export manifest, destroy allocation) when not necessary. Thus we would also avoid the flakiness which manifest export sometimes causes.